src/commands/app/deploy.js

```diff
@@ -324,6 +342,10 @@ Deploy.flags = {
     default: true,
     exclusive: ['action']
   }),
+  'force-deploy': flags.boolean({
+    description: '[default: false] Force deploy extensions regardless of published status or workspace',
```

Comment:
What about `Force deploy changes, regardless of production Workspace being published in Exchange.`

Reply:
Sounds good to me 👍